### PR TITLE
Escape room 3

### PR DESCRIPTION
**Pull request by drbalar**

### CODE REVIEW

**Review by ahmedhamodi — COMMENTED** [review]
No issues from me on code perspective. Might be good to see a live demo together tho!

File: scripts/mockdata/scenes_e3s1.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "name": "Classroom - Scene 1",
+    "description": "Supporting a Youth in Crisis - Room 1 - Check in with your classmates",
```
Comment (ahmedhamodi) [inline scripts/mockdata/scenes_e3s1.json:4]: The descriptions for the rooms are not updated

File: scripts/mockdata/objects_e3s5.json [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "text-pane",
+        "jsonData": {
+          "data": [
+            {
+              "text": "You check in with your CCC (supervisor on shift) introduce yourself and let them know that this is your first contact."
+            },
+            {
+              "text": "They welcome you to the team and ensure you that they will hop on an monitor right away for extra support."
```
Comment (ahmedhamodi) [inline scripts/mockdata/objects_e3s5.json:145]: "hop on a monitor"

**Review by ahmedhamodi — COMMENTED** [review]
Two things to update + changing the image to have numbers

File: app/routes/ui/admin_scene.py [hunk 3]
```diff
@@ -3,10 +3,10 @@
 from config import config
 from library.postgres import get_scene_from_postgres
 from models import get_session
-from routes.base import BaseAdminUIHandler
+from routes.base import BaseUIHandler
 
 
-class UIAdminSceneHandler(BaseAdminUIHandler):
+class UIAdminSceneHandler(BaseUIHandler):
```
Comment (ahmedhamodi) [inline app/routes/ui/admin_scene.py:9]: Intended change?

**Review by ahmedhamodi — APPROVED** [review]
LGTM, minus one comment

File: app/routes/ui/admin_scene.py (hunk 3 above)
Comment (drbalar) [reply app/routes/ui/admin_scene.py:9]: ooh no no

**Review by ahmedhamodi — APPROVED** [review]

**Pull request merged**